lxd/network_allocations.go

```diff
@@ -126,7 +126,12 @@ func networkAllocationsGet(d *Daemon, r *http.Request) response.Response {
 
 	result := make([]api.NetworkAllocations, 0)
 
-	userHasPermission, err := s.Authorizer.GetPermissionChecker(r.Context(), auth.EntitlementCanView, entity.TypeNetwork)
+	canViewNetwork, err := s.Authorizer.GetPermissionChecker(r.Context(), auth.EntitlementCanView, entity.TypeNetwork)
+	if err != nil {
+		return response.SmartError(err)
+	}
+
+	canViewInstanceIgnoringEffectiveProject, err := s.Authorizer.GetPermissionCheckerWithoutEffectiveProject(r.Context(), auth.EntitlementCanView, entity.TypeInstance)
```

Comment:
It'd be good to have a comment here explaining why we are using this.

Reply:
Ah yeah I'll reword the comment from the commit message.